include/zephyr/xen/events.h

```diff
@@ -22,6 +23,7 @@ typedef struct event_channel_handle evtchn_handle_t;
 void notify_evtchn(evtchn_port_t port);
 int bind_event_channel(evtchn_port_t port, evtchn_cb_t cb, void *data);
 int unbind_event_channel(evtchn_port_t port);
+int get_missed_events(evtchn_port_t port);
```

Comment:
Also I think, that user should not care about missed event. Probably, we need just call assigned event channel callback after it will be binded by user (in case the `events_missed` is equal to `true` during binding). It should happens in `bind_event_channel`, not in separate function.

Reply:
Yeah my first implementation was like that, but it was changed after internal review, now I have some leverage to support it, I will push it soon, also including `bool events_missed` to `struct event_channel_handle`